src/kOS/Screen/Interpreter.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
I have no idea why git seems to think this line changed.  It didn't.  It's literally identical, even down to the whitespace.

Reply:
A lot of files have an utf byte order mark (BOM) in front of it. Sometimes used to happen when creating new files with older VS versions.